src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -1249,6 +1306,16 @@ def to_json(self) -> str:
         created = datetime_to_str(self.created)
         return f'{created} {self.kind_subject()} [{self.level}] "{self.message}"'
 
+    def to_dict(self) -> dict:
+        # Convert events data to dict.
+        events = {}
+        created = datetime_to_str(self.created)
+        events['created'] = created
+        events['subject'] = self.kind_subject()
+        events['level'] = self.level
+        events['message'] = self.message
+        return events
```

Comment:
I'd say the same here:
```suggestion
        return {
          'created': datetime_to_str(self.created),
          'subject': self.kind_subject(),
          ....
        }
```

Reply:
Makes sense.! Thanks